websites/B/BloomHost/dist/metadata.json

```
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "author": {
+    "name": "BossDaily",
+    "id": "274973338676494347"
+  },
+  "service": "BloomHost",
+  "description": {
+    "en": "Bloom offers affordable dedicated Minecraft server hosting and dedicated Virtual Private Servers. Experience the outstanding performance of Bloom today."
+  },
+  "url": "bloom.host",
+  "regExp": "(?:[a-z0-9]+\\.)*bloom\\.host",
```

you should add the supported URLs to the URL property and not use a regex

---

I literally added all the urls but was told to use regexp instead

---

And you yourself marked that comment as resolved

---

When I changed to regex

---

https://github.com/PreMiD/Presences/pull/6094/files/10d90c00468d3e332b04e146f7f18199dee92061#diff-f4e68cb20a0be60bae84e37a21aee38c700ca4772fde8cd22f81c83e0492b668

---

I believe I marked that as resolved as a way to disregard that comment but I'm not sure. I will discuss this internally but I believe you should either specify the words supported in the regex or put the URLs in the array.

---

can I resolve this?

---

Remove the numbers in the regex

---

just this then, we're going with the regex
```suggestion
  "regExp": "(?:[a-z]+\\.)*bloom\\.host",
```